managed/ui/src/components/config/Storage/StorageConfigurationContainer.js

```diff
@@ -17,6 +17,20 @@ import {
 import { openDialog, closeDialog } from '../../../actions/modal';
 import { toast } from 'react-toastify';
 
+const customerConfigToasterHandler = (errorMessageObject) => {
+  errorMessageObject instanceof Object
```

Comment:
For this condition you can use "isNonEmptyObject" for checking errorMessageObject from "ObjectUtils" .

Reply:
Changes looks good , just check my above comment .

Reply:
Done, Thanks